src/Library/demos/Box/main.js

```diff
@@ -0,0 +1,76 @@
+import Gtk from "gi://Gtk";
+
+const interactive_box = workbench.builder.get_object("interactive_box");
+const button_add = workbench.builder.get_object("button_add");
+const prepend = workbench.builder.get_object("prepend");
+const button_remove = workbench.builder.get_object("button_remove");
+var count = 0;
```

Comment:
```suggestion
let count = 0;
```

`var` isn't bad, but it is usually avoided in GJS unless the variable really needs to be hoisted.